manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -148,3 +148,14 @@ spec:
         for: 1h
         labels:
           severity: info
+      - alert: APIRemovedInNextEUSReleaseInUse
+        annotations:
+          message: >-
+            Deprecated API that will be removed in the next EUS version is being used. Removing the workload that is using
+            the {{"{{$labels.group}}"}}.{{"{{$labels.version}}"}}/{{"{{$labels.resource}}"}} API might be necessary for
+            a successful upgrade to the next EUS cluster version. Refer to the audit logs to identify the workload.
+        expr: |
+          group(apiserver_requested_deprecated_apis{removed_release="1.23"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total[4h]))) > 0
```

Comment:
you actually need everything removed from 1.19 to 1.23, right?

Reply:
Updated